fab/fabfile.py

```diff
@@ -692,6 +694,14 @@ def copy_tf_localsettings():
         ))
 
 
+@task
+@roles(ROLES_TOUCHFORMS)
+def build_formsplayer():
+    spring_dir = '{}/{}'.format(env.code_root, 'submodules/formsplayer')
+    with cd(spring_dir):
+        sudo('./gradlew build')
```

Comment:
good points. so when i discussed with will he said that the build would take around ~30 seconds.  did seem horrible, but agree it is adding to an already long deploy. i don't know too much about jenkins' builds, but could talk to phillip about that. i also considered moving this completely outside the realm of HQ, but then realized that'd take a some amount of deploy rework. i think we'd have to reimplement the symlinking etc. if we think it's worth it, it's definitely possible